cmd/catalog.go

```diff
@@ -64,12 +64,12 @@ var catalogListComponentCmd = &cobra.Command{
 
 var catalogListServiceCmd = &cobra.Command{
 	Use:   "services",
-	Short: "Lists all the services from service catalog",
-	Long:  "Lists all the services from service catalog",
+	Short: "Lists all services available",
+	Long:  "Lists all services available",
 	Example: `  # List all services
   odo catalog list services
 
- # Search for a supported services
+ # Search for a supported a service
   odo catalog search services mysql
```

Comment:
Should we use the plural or singular form ? If the odo's command can do a search to retrieve all the services containing the word passed as parameter, then go ahead to use the plural `services` word otherwise that should be the `singular` -> `odo catalog search service mysql`